kf/bpf_test.go

```diff
@@ -571,19 +582,25 @@ func Test_PutNextProgFDFromID(t *testing.T) {
 		//		Pid          int
 		FilePath     string
 		RestartCount int
+		hostConfig   *config.Config
 	}
 	tests := []struct {
-		name    string
-		fields  fields
-		wantErr bool
-		progId  int
+		name       string
+		fields     fields
+		wantErr    bool
+		progId     int
+		hostConfig *config.Config
 	}{
 		{
 			name: "emptyMapName",
 			fields: fields{
 				Program: models.BPFProgram{
 					MapName: "",
 				},
+				hostConfig: &config.Config{
+					BpfMapDefaultPath:  "/sys/fs/bpf",
+					TcMapsRelativePath: "/tc/globals",
```

Comment:
Some files have "tc/globals" and some have "/tc/globals".  Why the inconsistency?

Reply:
Suggestion during TSC meeting: delete TcMapsRelativePath and put "tc/globals" in the map name in the programs themselves

Reply:
done